modules/stats.php

```diff
@@ -594,7 +595,7 @@ function stats_reports_page( $main_chart_only = false ) {
 					 */
 					apply_filters( 'jetpack_static_url', "{$http}://en.wordpress.com/i/loading/loading-64.gif" )
 				); ?>" /></p>
-		<p style="font-size: 11pt; margin: 0;"><a href="https://wordpress.com/stats/<?php echo esc_attr( $domain ); ?>" target="_blank"><?php esc_html_e( 'View stats on WordPress.com right now', 'jetpack' ); ?></a></p>
+		<p style="font-size: 11pt; margin: 0;"><a href="<?php echo esc_url( $stats_url ); ?>" target="_blank"><?php esc_html_e( 'View stats on WordPress.com right now', 'jetpack' ); ?></a></p>
```

Comment:
Let's add `rel="noopener noreferrer"` here as well.

Reply:
done in 9908ab4e06da5580156266a559a8f963b56c01df